solr_conf/conf/solrconfig.xml

```diff
@@ -11,10 +11,10 @@
   <directoryFactory name="DirectoryFactory" class="${solr.directoryFactory:solr.StandardDirectoryFactory}"/>
 
   <!-- solr lib dirs -->
```

Comment:
Are you sure about that? Don't we shared solrs with other apps (does that mean we sharing configs)?  Are you sure ansible not overwritting them? etc? Sure be nice to have some documentation around these things. Couldn't find anything relating to this in `ansible-config`. And ansible-dev/local dev use jetty's solr config.